src/topology/algebra/ordered.lean

```diff
@@ -1944,3 +1944,23 @@ lemma tendsto_of_monotone {ι α : Type*} [preorder ι] [topological_space α]
   tendsto f at_top at_top ∨ (∃ l, tendsto f at_top (𝓝 l)) :=
 if H : bdd_above (range f) then or.inr ⟨_, tendsto_at_top_csupr h_mono H⟩
 else or.inl $ tendsto_at_top_at_top_of_monotone' h_mono H
+
+lemma tendsto_neg_nhds_within_Ioi {α : Type*} [ordered_add_comm_group α]
```

Comment:
@jcommelin Should we have a multiplicative version of this? I don't know if we're going to have any topological `ordered_comm_group`s.

Reply:
If yes, then `@[to_additive]` should be able to transfer this lemma from multiplicative to additive.

Reply:
It doesn't cost anything, so I made it anyway